tests/components/fritz/test_button.py

```diff
@@ -73,3 +76,127 @@ async def test_buttons(
 
         button = hass.states.get(entity_id)
         assert button.state != STATE_UNKNOWN
+
+
+async def test_wol_button(
+    hass: HomeAssistant,
+    fc_class_mock,
+    fh_class_mock,
+) -> None:
+    """Test Fritz!Tools wake on LAN button."""
+    entry = MockConfigEntry(domain=DOMAIN, data=MOCK_USER_DATA)
+    entry.add_to_hass(hass)
+
+    with patch(
+        "homeassistant.helpers.entity.Entity.entity_registry_enabled_default",
```

Comment:
We have a fixture for this.

https://github.com/home-assistant/core/blob/f91994d788305436f6998f5d68cdfcee29a2ab88/tests/components/conftest.py#L40-L47